src/mps/abstractmps.jl

```diff
@@ -1605,13 +1605,14 @@ end
 
 function truncate!(::Algorithm"frobenius", M::AbstractMPS; kwargs...)
```

Comment:
For keyword arguments that aren't getting forwarded, I prefer the style of putting them directly in the function signature, i.e:
```julia
function truncate!(::Algorithm"frobenius", M::AbstractMPS; site_range=eachindex(M), kwargs...)
  [...]
end
```
instead of using `get(kwargs, [...])`.

Reply:
Looks good, thanks!